test/0.8.9/withdrawal-queue.test.js

```diff
@@ -1,30 +1,34 @@
 const { artifacts, contract } = require('hardhat')
 const { bn } = require('@aragon/contract-helpers-test')
-const { assertBn, assertRevert } = require('@aragon/contract-helpers-test/src/asserts')
+const { assertBn } = require('@aragon/contract-helpers-test/src/asserts')
 const { assert } = require('chai')
-const { ETH } = require('../helpers/utils')
+const { ETH, StETH, shareRate } = require('../helpers/utils')
+const { assertRevert } = require('../helpers/assert')
 
 const WithdrawalQueue = artifacts.require('WithdrawalQueue.sol')
-const Owner = artifacts.require('Owner.sol')
-const StETHMock = artifacts.require('StETHMockForWithdrawalQueue.sol')
+const StETHMock = artifacts.require('StETHMock.sol')
 const WstETH = artifacts.require('WstETH.sol')
 const OssifiableProxy = artifacts.require('OssifiableProxy.sol')
 
-contract('WithdrawalQueue', ([recipient, stranger, daoAgent]) => {
-  let withdrawal, withdrawalImpl, owner, steth, wsteth
+contract('WithdrawalQueue', ([recipient, stranger, daoAgent, user]) => {
+  let withdrawal, withdrawalImplAddress, steth, wsteth
 
   beforeEach('Deploy', async () => {
-    owner = (await Owner.new({ value: ETH(300) })).address
-    await ethers.provider.send('hardhat_impersonateAccount', [owner])
-    steth = await StETHMock.new()
+    steth = (await StETHMock.new({ value: ETH(600) }))
+    await ethers.provider.send('hardhat_impersonateAccount', [steth.address])
+    steth.setTotalPooledEther(ETH(300))
+
     wsteth = await WstETH.new(steth.address)
 
-    withdrawalImpl = (await WithdrawalQueue.new(owner, steth.address, wsteth.address)).address
-    console.log({ withdrawalImpl })
-    const withdrawalProxy = await OssifiableProxy.new(withdrawalImpl, daoAgent, '0x')
+    withdrawalImplAddress = (await WithdrawalQueue.new(steth.address, steth.address, wsteth.address)).address
+    const withdrawalProxy = await OssifiableProxy.new(withdrawalImplAddress, daoAgent, '0x')
     withdrawal = await WithdrawalQueue.at(withdrawalProxy.address)
+
     await withdrawal.initialize(daoAgent)
     await withdrawal.resumeRequestsPlacement({ from: daoAgent })
+
+    steth.mintShares(user, 1)
+    steth.approve(withdrawal.address, StETH(300), { from: user })
   })
 
   context('Enqueue', async () => {
```

Comment:
Should check that by-default the queue is paused (can't enqueue requests)

Reply:
Yep, but here, we are just restoring the old tests, we are to write many more tests for the newer functions later, of cause.